app/controllers/memberships_controller.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+class MembershipsController < ApplicationController
```

Comment:
Add custom action for getting memberships for a singular user. Reserve index for admin and organization leader management actions.